cmd/state-handler/Dockerfile

```diff
@@ -4,10 +4,10 @@ COPY . .
 RUN CGO_ENABLED=0 GOOS=linux go build -o /bin/state-handler github.com/nmstate/kubernetes-nmstate/cmd/state-handler
 
 FROM centos:7
-RUN yum-config-manager --add-repo https://copr.fedorainfracloud.org/coprs/nmstate/nmstate-el7/repo/epel-7/nmstate-nmstate-el7-epel-7.repo && \
-    yum -y install epel-release && \
+RUN yum -y install epel-release && \
     yum -y update && \
-    yum -y install nmstate && \
-    yum clean all
+    yum -y install \
+        https://kojipkgs.fedoraproject.org//packages/nmstate/0.0.5/1.el7/noarch/python2-libnmstate-0.0.5-1.el7.noarch.rpm \
+        https://kojipkgs.fedoraproject.org//packages/nmstate/0.0.5/1.el7/noarch/nmstate-0.0.5-1.el7.noarch.rpm && \
```

Comment:
you need to add the `yum clean all` or remove the `&& \` at the end of the line the build failed